examples/librispeech/librispeech.ipynb

```diff
@@ -231,31 +240,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [],
    "source": [
-    "train_dl = torch.utils.data.DataLoader(cuts_train, batch_size=32, shuffle=True)\n",
-    "dev_dl = torch.utils.data.DataLoader(cuts_dev, batch_size=32)"
+    "train_dl = torch.utils.data.DataLoader(dataset_train, batch_size=32, shuffle=True)\n",
```

Comment:
I think there is no point in building a dataloader from a `K2SpeechRecognitionDataset` without specifying a sampler.

The resulting dataloader is useless since `__getitem__` of `K2SpeechRecognitionDataset`
accepts a list of `str`, not a single `int`.